modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -93,17 +93,23 @@ export interface TransactionExplanation extends BaseTransactionExplanation<strin
   signatures: number;
 }
 
+export interface TransactionInfo {
+  txHexes?: Record<string, string>;
```

Comment:
A comment here might be good to explain what the tx hexes are being mapped to. It looks like that's txIds based on the code below altho I'm also a bit confused why there multiple txIds for one transaction - these would be transactions from other chains potentially I'm guessing?

Reply:
will add comment